build.gradle

```diff
@@ -25,13 +25,13 @@ allprojects {
 
 ext {
     // Sdk and tools
-    minSdkVersion = 11
-    targetSdkVersion = 23
-    compileSdkVersion = 23
-    buildToolsVersion = '23.0.2'
+    minSdkVersion = 16
```

Comment:
Why this specific version ? Is this upgrade required ?

Reply:
It is not required but I think that will be better if min sdk version for `app` and for `authdemo` modules will be set up in one place

Reply:
My main point was that it would be nice to support as many Androids versions as possible and not upgrade the minimal version when not required.

Reply:
Oh, I see

Reply:
I have seen and in `app` and `authdemo` was used minSdkVersion 16 (const `minSdkVersion` was not used)

Reply:
[authdemo](https://github.com/xwiki-contrib/android-authenticator/pull/12/files/3ef222e192ecf9b8d1f7436c6a8079e7b9662c7d#diff-2aded2f463db7cb514509f470c0f2b13L8)
[app](https://github.com/xwiki-contrib/android-authenticator/pull/12/files/3ef222e192ecf9b8d1f7436c6a8079e7b9662c7d#diff-39e7d8c00954e920b98e7636f0ac30b2L13)

Reply:
OK then :)